src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -383,6 +486,21 @@ public String getLocation() {
       return location;
     }
 
+    @Nullable
+    @Description("Gets the value of content type")
```

Comment:
this should be a javadoc not annotation. The javadoc should include mapping of format -> content type

Reply:
PR updated.